<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Lunabotics</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="positionTitleLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QLabel" name="posXTitleLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
   </widget>
   <widget class="QLabel" name="posYTitleLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="posXLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="posYLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="orientationTitleLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Orientation</string>
    </property>
   </widget>
   <widget class="QLabel" name="orientationLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>theta</string>
    </property>
   </widget>
   <widget class="Line" name="leftSeparatorLine">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>16</width>
      <height>421</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="driveForwardLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>W</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="driveLeftLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="driveBackLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>S</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="driveRightLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>D</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="connectionBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QPushButton" name="lockButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>114</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Unlock</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="portLineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>5556</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ipLineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>192.168.218.129</string>
     </property>
    </widget>
    <widget class="QLabel" name="ipTitleLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP address</string>
     </property>
    </widget>
    <widget class="QLabel" name="portTitleLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="driveTitleLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Drive:</string>
    </property>
   </widget>
   <widget class="QLabel" name="telemetryTitleLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Telemetry:</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="controlModesTabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabAckermann">
     <attribute name="title">
      <string>Ackermann</string>
     </attribute>
     <widget class="QCheckBox" name="ackermannLinearSpeedCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Limit linear speed to                         km/h</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ackermannDependentValueCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Limit wheel rotation angle to                     Â°</string>
      </property>
     </widget>
     <widget class="QPushButton" name="useAckermannButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Use this mode</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ackermannLinearSpeedEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ackermanDependentValueEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSpot">
     <attribute name="title">
      <string>Spot</string>
     </attribute>
     <widget class="QCheckBox" name="spotDependentValueCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Limit rotational speed to                    km/h</string>
      </property>
     </widget>
     <widget class="QPushButton" name="useSpotButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Use this mode</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="spoeLinearSpeedCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Limit linear speed to                         km/h</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="spotLinearSpeedEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="spotDependentValueEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabLateral">
     <attribute name="title">
      <string>Lateral</string>
     </attribute>
     <widget class="QCheckBox" name="lateralDependentValueCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Limit lateral speed to                     km/h</string>
      </property>
     </widget>
     <widget class="QPushButton" name="useLateralButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Use this mode</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="lateralLinearSpeedCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Limit linear speed to                         km/h</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lateralLinearSpeedEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lateralDependentValueEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="controlModeTitleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Control Modes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="roverViewLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>221</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="LunaGUIResources.qrc">:/images/rover.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="rightSeparatorLine">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>16</width>
      <height>421</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="mapTitleLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Map:</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="mapView">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>50</y>
      <width>301</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="autonomyCheckbox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual Control</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLunaGUI">
    <property name="title">
     <string>LunaGUI</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuLunaGUI"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LunaGUIResources.qrc"/>
 </resources>
 <connections/>
</ui>
